### ISSUE TITLE
Snapshots lose edits to molecule rendering

### ISSUE THREAD

**Issue by phraenquex** [m2ms/fragalysis-frontend#1145]

**Comment by boriskovar-m2ms** [comment]
This means that the NGL settings are changed for respective objects (like representation etc.) are not preserved in the snapshot?

**Comment by phraenquex** [comment]
Yes.  Stuff I changed under "Display controls" didn't persist when I loaded the snapshot.

They also disappear if I turn off the molecule and turn it on again.  Most users would consider this unintuitive/frustrating - but I don't know if it's easy to fix, certainly within a "setting".  

Undo/redo behaves very unpredictably:  
* change a ligand's representation to "surface"
* turn off the ligand, turn it back on
* click on undo a few times
* click on redo a few times
It certainly doesn't go forwards and backwards - so there's a bug there, and I assume it's related.

**Comment by phraenquex** [comment]
If it's not hard to do, then yes - that way the person looking at the snapshot, can figure out what's changed.

(In a future iteration, I'll be asking that some of these display modes directly from the LHS, e.g. on the drop-down, maybe from a new button.)

**Comment by matej-vavrek** [comment]
No it it hould not be hard to do. However I am not really sure what the right-click idea is, @phraenquex.

**Comment by phraenquex** [comment]
If it's not hard to do, then go ahead. 

We'll spec out the LHS-right-click idea separately - please remind me to create a place-holder ticket.

**Comment by phraenquex** [comment]
Restoration of edits from snapshots now moved to #1148.  

@matej-vavrek please deploy to test server.

**Comment by matej-vavrek** [comment]
@phraenquex, deployed. It should cover preserving state of display controls changes on objects.

**Comment by phraenquex** [comment]
Looks great!  Please get it through to staging and production as quick as possible.
